mta-sts.md

```diff
@@ -480,6 +480,40 @@ Recipients should also prefer to update the HTTPS policy body before updating
 the TXT record; this ordering avoids the risk that senders, seeing a new TXT
 record, mistakenly cache the old policy from HTTPS.
 
+## Policy Delegation
+
+Domain owners commonly delegate SMTP hosting to a different organization, such
+as an ISP or a Web host. In such a case, they may wish to also delegate the
+MTA-STS policy to the same organization which can be accomplished with two
+changes.
+
+First, the Policy Domain must point the `_mta-sts` record, via CNAME, to
+the `_mta-sts` record maintained by the hosting organization. This allows the
+hosting organization to control update signaling.
+
+Second, the Policy Domain must point the "well-known" policy location to the
+hosting organization. This can be done either by redirecting the `mta-sts` host to a
+host or CNAME specified by the hosting organization and by giving the hosting
+organization a TLS certificate which is valid for that host, or by setting up a
+"reverse proxy" (also known as a "gateway") server that serves as the Policy
+Domain's policy the policy currently served by the hosting organization.
+
+For example, given a user domain `user.com` hosted by a mail provider
+`provider.com`, the following configuration would allow policy delegation:
+
+DNS:
+~~~
+_mta-sts.user.com.  IN CNAME _mta-sts.host.com.
+~~~
+
+Policy:
+~~~
+> GET /.well-known/mta-sts.txt
+> Host: mta-sts.user.com
+< HTTP/1.1 200 OK  # Response proxies content from https://mta-sts.host.com
```

Comment:
> https://mta-sts.host.com.

Change it to:
https://mta-sts.provider.com.

Reply:
Done.